src/main/kotlin/nexters/admin/service/auth/AuthDtos.kt

```diff
@@ -3,6 +3,11 @@ package nexters.admin.service.auth
 import nexters.admin.domain.user.Password
 import javax.validation.constraints.Email
 
+data class AdminLoginRequest(
+        val username: String,
+        val password: Password
```

Comment:
이 부분 고치기로 했으니, 아예 지금 바꿔버리죠!
```suggestion
        val password: String
```

Reply:
일단은 controller, service에서 굳이 Password 값 객체 대신 문자열로만 주고 받도록 수정해보았습니다.